src/production/__tests__/ComputedStatementImport.test.ts

```diff
@@ -0,0 +1,75 @@
+  it(`import('./atoms/' + a) should return a valid pattern`, () => {
+    // Tests the case:
+    // import('./atoms/' + b);
+    const result = convertComputedStatementToRegex(getAst(`'./atoms/' + a`));
+    expect(result).toEqual(['./atoms/', expressionValueTypes.SINGLE_ASTERISK]);
+  });
+
+  it(`import('./atoms/' + b + '/' + c) should return a valid pattern`, () => {
+    // Tests the case:
+    // import('./atoms/' + b + '/' + c);
+    const result = convertComputedStatementToRegex(getAst(`'./atoms/' + b + '/' + c`));
+    expect(result).toEqual(['./atoms/', expressionValueTypes.DOUBLE_ASTERISK, '/', expressionValueTypes.SINGLE_ASTERISK]);
+  });
+
+  it(`import(b + '/' + c) should fail and return an empty array`, () => {
+    // mocking console.error and catch the output
+    const originalError = console.error
+    let consoleOutput = []
+    const mockedError = output => consoleOutput.push(output)
+    console.error = mockedError
+
+    // Tests the case:
+    // import(b + '/' + c);
+    const result = convertComputedStatementToRegex(getAst(`b + '/' + c`));
+    expect(result).toEqual([]);
+    expect(consoleOutput).toEqual(["You're computed import needs to start with a string! i.e. './'"]);
+
+    console.error = originalError;
+  });
+
+  it('import(`./atoms/${a}`) should return a valid pattern', () => {
+    // Tests the case:
+    // import(`./atoms/${a}`);
+    const result = convertComputedStatementToRegex(getAst('`./atoms/${a}`'));
+    expect(result).toEqual(['./atoms/', expressionValueTypes.SINGLE_ASTERISK]);
+  });
+
+  it('import(`./atoms/${b}/${c}`) should return a valid pattern', () => {
+    // Tests the case:
+    // import(`./atoms/${b}/${c}`);
+    const result = convertComputedStatementToRegex(getAst('`./atoms/${b}/${c}`'));
+    expect(result).toEqual(['./atoms/', expressionValueTypes.DOUBLE_ASTERISK, '/', expressionValueTypes.SINGLE_ASTERISK]);
+  });
+
+  it('import(`./atoms/${b}${c}`) should return a valid pattern', () => {
+    // Tests the case:
+    // import(`./atoms/${b}${c}`);
+    const result = convertComputedStatementToRegex(getAst('`./atoms/${b}${c}`'));
+    expect(result).toEqual(['./atoms/', expressionValueTypes.SINGLE_ASTERISK]);
+  });
+
+  it('import(`./atoms/${b}${c}/${d}`) should return a valid pattern', () => {
+    // Tests the case:
+    // import(`./atoms/${b}${c}`);
+    const result = convertComputedStatementToRegex(getAst('`./atoms/${b}${c}/${d}`'));
+    expect(result).toEqual(['./atoms/', expressionValueTypes.DOUBLE_ASTERISK, '/', expressionValueTypes.SINGLE_ASTERISK]);
+  });
+
+  it('import(`./atoms/${b + \'/\' + c}`) should return a valid pattern', () => {
+    // Tests the case:
+    // import(`./atoms/${b + '/' + c}`);
+    const result = convertComputedStatementToRegex(getAst("`./atoms/${b + '/' + c}`"));
+    expect(result).toEqual(['./atoms/', expressionValueTypes.DOUBLE_ASTERISK, '/', expressionValueTypes.SINGLE_ASTERISK]);
+  });
+});

```

Comment:
I would add tests for non-supported nodes, like function calls and such.

Reply:
consider it done